src/styles/components/button.scss

```diff
@@ -23,7 +23,8 @@
     display: flex;
     font-size: 14px;
     justify-content: center;
-    padding: 12px 24px;
+    height: 36px;
+    padding: 0 12px;
```

Comment:
I think we still need to use padding. There's always the possibility that a button will wrap when we're not expecting it to with a fixed height and no padding on top/bottom that's going to look bad.